src/sagemaker_containers/cli/serve.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the 'license' file accompanying this file. This file is
+# distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import sagemaker_containers
+
+
+def main():
+    env = sagemaker_containers.environment.ServingEnvironment()
+    sagemaker_containers.server.start_server(env.flask_app)
```

Comment:
I would rename flask_app to framework_module or app_module.

The reason is that gunicorn runs more things than just Flask http://docs.gunicorn.org/en/stable/run.html

Reply:
will do